app/models/compliance.rb

```diff
@@ -83,7 +83,7 @@ def self.check_compliance(target, _inputs = {})
       return
     end
 
-    compliance_result = results[:actions].nil? ? true : !results[:actions].key?(:compliance_failed)
+    compliance_result = results[:action].nil? ? true : !results[:action].key?(:compliance_failed)
```

Comment:
@bzwei Please review.  I doesn't look like you changed this from actions -> action in the links above intentionally.

@moolitayer Please put the git sha e5f24705b8f8b2b5 in the commit message and description as reference for when this broke.